cynic-querygen/Cargo.toml

```diff
@@ -18,6 +18,7 @@ once_cell = "1.9"
 rust_decimal = "1.22"
 thiserror = "1.0.30"
 uuid = { version = "1", features = ["v4"] }
+async-graphql = { version = "7" }
```

Comment:
async-graphql is a very heavy dependency to pull in, particularly just for a single 3 variant enum.  Can you implement this without the dependency please.